.maestro/notifications_permissions_android13_plus/2_-_permissions_denied.yaml

```diff
@@ -0,0 +1,12 @@
+appId: com.duckduckgo.mobile.android
```

Comment:
don't you want to assert that the Notify Me component is visible in Downloads? I don't see much value on this test if this is not doing anything else than tapping the notifications popup.

Reply:
Yes, good catch! Will update :+1:

Reply:
Updated :+1: